src/os/bluestore/BlueStore.cc

```diff
@@ -18299,7 +18323,34 @@ void BlueStore::_log_alerts(osd_alert_list_t& alerts)
   } else if (!spillover_alert.empty()){
     spillover_alert.clear();
   }
-
+  if (cct->_conf->bluestore_slow_op_event_counter_threshold) {
+    _trim_slow_op_event_queue(mono_clock::now());
```

Comment:
IMO more elegant code would be:
auto qsize = _trim_slow_op_event_queue();
if (qsize) {
...
}